docs/sphinx/source/whatsnew/v0.8.0.rst

```diff
@@ -17,6 +17,8 @@ API Changes
 ~~~~~~~~~~~
 * Removed ``run_parallel_calculations`` and ``n_workers_for_parallel_calcs``
   from :py:func:`pvlib.bifacial.pvfactors_timeseries` inputs (:issue:`902`)(:pull:`934`)
+* :py:func:`pvlib.iotools.read_tmy3` can now only read local data files
```

Comment:
Would be good to add a reason for this so that a finger is pointed towards a mesa in Colorado instead of other locations in the Southwest.

Reply:
Also a suggestion for what to use instead.